src/Language/Fixpoint/Solver/Sanitize.hs

```diff
@@ -160,11 +161,11 @@ eliminateEta cfg si
       | otherwise
       = Nothing
 
-theoryEnv :: Config -> F.GInfo c a -> F.SEnv F.TheorySymbol
-theoryEnv cfg si
-  =  Thy.theorySymbols (Cfg.solver cfg)
-  <> Thy.theorySymbols (F.defns si)
-  <> Thy.theorySymbols (F.ddecls si)
+-- theoryEnv :: Config -> F.GInfo c a -> F.SEnv F.TheorySymbol
+-- theoryEnv cfg si
+--   =  Thy.theorySymbols (Cfg.solver cfg)
+--   <> Thy.theorySymbols (F.defns si)
+--   <> Thy.theorySymbols (F.ddecls si)
```

Comment:
```suggestion
```

This comment can be removed, I guess. `theoryEnv` is moved to the SortCheck module.